_source/_posts/2022-01-04-angular-auth0-quickly.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: blog_post
+title: "Add OpenID Connect to Angular Apps Quickly"
+author: matt-raible
+by: advocate
+communities: [javascript]
+description: "This tutorial shows you how to add OpenID Connect authentication with Auth0 in just a few minutes."
+tags: [javascript, angular, typescript, auth0]
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
+github: https://github.com/oktadev/auth0-angular-example
+---
+
+AngularJS 1.0 was released in October 2020. At the time, it was one of the most popular web frameworks to ever see the light of day. Developers loved it and many apps were created with it. It's next major release, Angular 2.0, was released in September 2016. Six years is a long time between major releases. Angular 2 took over two years to develop and many folks jumped to other frameworks rather than waiting for the rewrite. 
```

Comment:
```suggestion
AngularJS 1.0 was released in October 2010. At the time, it was considered one of the most revolutionary and popular web frameworks to ever see the light of day. Developers loved it and many apps were created with it. As a pioneer in the JS framework space, however, AngularJS had some growing pains and major issues. The team went back to the drawing board for a major breaking release with Angular 2. It took 2 years to develop, and influenced many devs to hop onto other frameworks in the meantime.  
```

I love remembering the heyday of Angular JS - Is this an effective hook for the rest of the blog post?

Reply:
It's not as much of a hook as an intro. I still find a lot of people confuse Angular JS and Angular, so it's an attempt to mitigate that.